pkg/index/job/correction/config/config.go

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package setting stores all server application settings
+package config
+
+import (
+	"github.com/vdaas/vald/internal/config"
+	"github.com/vdaas/vald/internal/errors"
+)
+
+type GlobalConfig = config.GlobalConfig
+
+// Data represents a application setting data content (config.yaml).
+// In K8s environment, this configuration is stored in K8s ConfigMap.
+type Data struct {
+	config.GlobalConfig `json:",inline" yaml:",inline"`
+
+	// Server represent all server configurations
+	Server *config.Servers `json:"server_config" yaml:"server_config"`
+
+	// Observability represent observability configurations
+	Observability *config.Observability `json:"observability" yaml:"observability"`
+
+	// Indexer represent agent auto indexing service configuration
+	Corrector *config.Corrector `json:"corrector" yaml:"corrector"`
+}
+
```

Comment:
Could you please add a comment?

Reply:
[5a73044](https://github.com/vdaas/vald/pull/2194/commits/5a73044f8586e5ea087c9df0d289388e1cb14b69)